mono/mini/debugger-agent.c

```diff
@@ -869,7 +869,7 @@ parse_flag (const char *option, char *flag)
 	else if (!strcmp (flag, "n"))
 		return FALSE;
 	else {
-		fprintf (stderr, "debugger-agent: The valid values for the '%s' option are 'y' and 'n'.\n", option);
+		g_warning ("debugger-agent: The valid values for the '%s' option are 'y' and 'n'.\n", option);
 		exit (1);
```

Comment:
g_warning () is used to signal some kind of low level error condition, while fprintf(stderr is used to print an error message to the user, so the two are not the same.

Reply:
fprintf won't log to logcat on android. Ludovic advised I used g_warning instead. Should I use g_notify perhaps?

Reply:
Use g_printerr i think.

Reply:
We should use `g_printerr`. That will involve calling `mono_trace_set_print_handler` and `mono_trace_set_printerr_handler` at https://github.com/alexanderkyte/mono/blob/155acf82681d665637776b8ccad5247dd036ba10/sdks/android/jni/runtime-bootstrap.c#L376, but that could be done later and shouldn't block this PR.